content/design-systems-for-developers/react/en/test.md

```diff
@@ -37,87 +37,29 @@ Visual tests capture an image of every UI component in a consistent browser envi
 
 ![Visual test components](/design-systems-for-developers/component-visual-testing.gif)
 
-If you’re building a modern UI, visual testing saves your frontend team from time-consuming manual review and prevents expensive UI regressions. We’ll demo visual testing using Chromatic, an industrial-grade service by the Storybook maintainers.
+If you’re building a modern UI, visual testing saves your frontend team from time-consuming manual review and prevents expensive UI regressions.
 
-First, go to [chromatic.com](https://chromatic.com) and sign up with your GitHub account.
+In the <a href="https://www.learnstorybook.com/design-systems-for-developers/react/en/review/#publish-storybook">previous chapter</a> we learned how to publish Storybook using [Chromatic](https://www.chromatic.com/). We added a bold red border around each `Button` component and then requested feedback from teammates.
```

Comment:
```suggestion
In the <a href="review/#publish-storybook">previous chapter</a> we learned how to publish Storybook using [Chromatic](https://www.chromatic.com/). We added a bold red border around each `Button` component and then requested feedback from teammates.
```

I think we should use relative links?

Reply:
@jonniebigodes mentioned they didn't work quite right in Gatsby for some reason. I can confirm that I get a routing error when I use the relative link.

Reply:
I'm pretty sure we can make some kind of link that doesn't include a domain work

Reply:
@tmeasday the reason we went with including the full url for this two situations was due to the fact that Gatsby+ Markdown does not "like" that. When you add a link in markdown relative to another page with a heading with will bubble up to the top of the page. I've tested it and it will always default to the top of the page. 

We could go about this in two ways:

- Add the [gatsby-remark-autolink-headers plugin](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-remark-autolink-headers) to the site and we have a way use the links/navigation better.
- Manually update both `gatsby-ssr` and `gatsby-browser` like it's used in the plugin to include the navigation.

Me personally i'm more inclined to the first one. As at some point it could be extremely beneficial to include a TOC per chapter and proper heading linking, which will be extremely more beneficial for the translations.

A initial approach was put together [here](https://github.com/chromaui/learnstorybook.com/pull/299), per @domyen feedback i thought it over and it would be better to address this properly somewhere in the near future

Reply:
i've created a issue (#347) for tracking the usage of links in headings here. We can revisit it once we have a little more breathing room. 

Sounds good?

Reply:
OK!